pkg/models/item.go

```diff
@@ -71,14 +71,14 @@ func UnmarshalItem(item *api.Item) (proto.Message, error) {
 }
 
 // GetModelForItem returns model for given item.
-func GetModelForItem(item *api.Item) (RegisteredModel, error) {
+func GetModelForItem(item *api.Item) (KnownModel, error) {
```

Comment:
I think `RegisteredModel` fits more, but if you like it...

Reply:
Actually, in the situation when the models are retrieved from remote agent and not from local registry, it might be confusing to label them as registered.. 

To be honest, Im not certain either way..